doc/Installation/Installation-Ubuntu-1804-Nginx.md

```diff
@@ -85,75 +84,93 @@ systemctl restart mysql
 
 # Web Server
 
-## Configure and Start PHP-FPM
+## Configure and Restart PHP-FPM
 
-Ensure date.timezone is set in php.ini to your preferred time zone.
-See <http://php.net/manual/en/timezones.php> for a list of supported
-timezones.  Valid examples are: "America/New_York",
-"Australia/Brisbane", "Etc/UTC".
-Please remember to set the system timezone as well.
+Ensure `date.timezone` is set in `php.ini` to your preferred time zone.
+
+Valid examples are: `"America/New_York"`, `"Australia/Brisbane"`, `"Europe/Bratislava"`, `"Etc/UTC"`.
+
+>See <http://php.net/manual/en/timezones.php> for a list of supported timezones and change `Europe/Bratislava` in sed command to your timezone.
 
 ```bash
-vi /etc/php/7.2/fpm/php.ini
-vi /etc/php/7.2/cli/php.ini
+echo date.timezone = \"Europe/Bratislava\" >> /etc/php/7.4/fpm/php.ini
+echo date.timezone = \"Europe/Bratislava\" >> /etc/php/7.4/cli/php.ini
```

Comment:
shouldn't  complete echo string be quoted?

Reply:
no it is good like it is. It just add new line with timezone